.github/workflows/examples.yml

```
@@ -283,25 +283,31 @@ jobs:
       - run: docker exec ${{ env.CONTAINER_NAME }} python3 -m pip config set global.index-url https://pypi.tuna.tsinghua.edu.cn/simple
       - name: Install onediff-quant if needed
         if: startsWith(matrix.image, 'onediff-pro')
         run: |
           docker exec ${{ env.CONTAINER_NAME }} python3 -m pip install --pre onediff-quant "numpy<2" -f https://oneflow-pro.oss-cn-beijing.aliyuncs.com/onediff-quant/
-      - name: Pip Install Requirements for ComfyUI & Test
+      - run: docker exec ${{ env.CONTAINER_NAME }} python3 -m pip install --user "huggingface_hub==0.25.0"
+      - run: docker exec ${{ env.CONTAINER_NAME }} python3 -m pip install -r /src/onediff/onediff_diffusers_extensions/examples/requirements.txt
+        if: matrix.test-suite == 'diffusers_examples'
+      - run: docker exec ${{ env.CONTAINER_NAME }} python3 -m pip install -r tests/comfyui/requirements.txt --user
+        if: matrix.test-suite == 'comfy'
+      - run: docker exec ${{ env.CONTAINER_NAME }} python3 -m pip install -r ComfyUI/requirements.txt --user"
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1009:info:1:1: The mentioned syntax error was in this simple command

(shellcheck)